Release 4.12.0 (#11296)
conda/conda: CHANGELOG.md

[//]: # (current developments)

## 4.12.0 (2022-03-XX)

### Enhancements

* Add support for libmamba integrations. (#11193)

  This is a new **experimental and opt-in** feature that allows use of the
  new [conda-libmamba-solver](https://github.com/conda-incubator/conda-libmamba-solver)
  for an improved user experience, based on the libmamba community project -
  the library version of the [mamba package manager](https://github.com/mamba-org/mamba).

>   This is an  which means you'll need to follow these steps:

Reviewer: ```suggestion
  Since the libmamba solver is provided as a standalone package you'll need to
  follow these **extra** steps to install & use the experimental solver:
```

Author: Wow, not sure what happened there with my English 🤣

>   1. Activate your conda base environment:
> 
>      ```
>      conda activate base
>      ```
> 
>      Then install the conda-libmamba-solver package:
> 
>      ```
>      conda install conda-libmamba-solver
>      ```
> 
>      Deactivate your base environment again
> 
>      ```
>      conda deactivate
>      ```

Reviewer: Can't this be combined into one?
```suggestion
  1. Install the `conda-libmamba-solver` package into your `base` environment:

     ```
     conda install -n base conda-libmamba-solver
     ```
```

  2. Try out the solver using the `--experimental-solver=libmamba` command line option.

     E.g. with a dry-run to install the ``scipy`` pacakge:

     ```
     conda create -n demo scipy --dry-run --experimental-solver=libmamba
     ```

     Or install in an activated conda environment:

     ```
     conda activate my-environment
     conda install scipy --experimental-solver=libmamba
     ```

> * Make sure that conda env update -f sets env vars from the referenced yaml file. (#10652)

Reviewer: ```suggestion
* Make sure that `conda env update -f` sets env vars from the referenced yaml file. (#10652)
```
